src/export-page/export-footer/ExportFooter.scss

```diff
@@ -0,0 +1,14 @@
+.export-footer-list {
+  list-style: none;
+  padding-left: 0;
+
+  li {
+    padding-bottom: 5px;
+    border-bottom: 1px solid #E5E5E5;
```

Comment:
create please variable for `src/assets/scss/_variables.scss`
with from `src/studio-header/header.scss` too

Reply:
Do we have the opportunity to use `$gray-100` [Paragon SCSS variable](https://paragon-openedx.netlify.app/foundations/colors)? Or maybe darken($white, desired percentage shading). It seems to me that after that we will not need to take out this color in a separate variable.